### PR TITLE
Add validation to check id column

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add validation to check id column
Related to https://github.com/acompany-develop/QuickMPC-libClient-py/pull/16
# Purpose
Check if the id columns match
Prevent requests with non-existent data_id
# Contents
- Fix send_share to save matching_column
- Add validation to check id column
- Add test

# Testing Methods Performed
CI

### CODE REVIEW

File: src/ManageContainer/Client/ManageToDb/Client.go [hunk 1]
```diff
@@ -281,3 +284,31 @@ func (c Client) GetElapsedTime(jobUUID string) (float64, error) {
 	path := fmt.Sprintf("%s/%s", resultDbPath, jobUUID)
 	return getElapsedTime(path)
 }
+
+
+func (c Client) GetMatchingColumn(dataID string) (int32, error) {
+	ls.Lock(dataID)
+	defer ls.Unlock(dataID)
+
```
Comment (KotaTakahashi9320) [inline src/ManageContainer/Client/ManageToDb/Client.go:292]:
```suggestion

```

File: src/ManageContainer/Server/LibcToManageContainer/L2m_test.go [hunk 2]
```diff
@@ -111,6 +114,27 @@ func TestExecuteComputation(t *testing.T) {
 	}
 }
 
+// id列が異なっていた場合にエラーがでるかテスト
+func TestExecuteComputationFailed(t *testing.T) {
```
Comment (mdonaka) [inline src/ManageContainer/Server/LibcToManageContainer/L2m_test.go:118]: @KotaTakahashi9320 
I would like the name of the test to be clear as to what is to be confirmed to fail.
`TestPredictFailed` as well.

Comment (KotaTakahashi9320) [reply src/ManageContainer/Server/LibcToManageContainer/L2m_test.go]: @mdonaka 
Thanks for the review!
Corrected test name
Please check it out

Comment (mdonaka) [reply src/ManageContainer/Server/LibcToManageContainer/L2m_test.go]: I think it's good! Thanks!

**Review by mdonaka — APPROVED** [review]

**Comment by mdonaka** [comment]
@KotaTakahashi9320 
Colud you rebase this branch?

**Comment by KotaTakahashi9320** [comment]
@mdonaka 
I rebased
Please check it.

**Pull request merged**